scripts/submit_emod3d.py

```diff
@@ -25,13 +25,19 @@ def write_sl_script(
     memory=const.DEFAULT_MEMORY,
     account=const.DEFAULT_ACCOUNT,
     machine=host,
+    nt=None,
 ):
     """Populates the template and writes the resulting slurm script to file"""
     workflow_config = load_config.load(
         os.path.dirname(os.path.realpath(__file__)), "workflow_config.json"
     )
 
-    set_runparams.create_run_params(srf_name, workflow_config=workflow_config)
+    if nt:
+        steps_per_checkpoint = int(nt/(60.0*convert_time_to_hours(run_time)) * 10)
```

Comment:
Spacing and ordering.

10 minutes should be a constant. i.e. CHECKPOINT_DURATION